apps/yapms/src/lib/utils/loadMap.ts

```diff
@@ -80,4 +113,140 @@ function loadFromJson(mapData: unknown): void {
 	RegionsStore.set(regionsStoreUpdated);
 }
 
-export { loadFromJson, loadFromFile };
+function convertTCTtoYapms(tct: unknown) {
+	const parsedData = FileSchema_TCT.safeParse(tct);
+	if (parsedData.success === false) {
+		return;
+	}
+
+	const yapmsData = {
+		map: {
```

Comment:
Can we give some sort of feedback to the user if they try to import a map that is not a US map? For instance, this is a Netherlands map:
[tct_nederland.json](https://github.com/user-attachments/files/20746145/tct_nederland.json)

I looked into the schema and there doesn't seem to be a way to do this, which is fine, it's no big deal. Leaving this here still.

Reply:
Good idea, added a loop to verify that all the regions for the map exist in the list of states.